python/lsst/daf/butler/datastores/genericDatastore.py

```diff
@@ -148,7 +148,7 @@ def _move_to_trash_in_registry(self, ref: DatasetRef) -> None:
         # Note that a ref can point to component dataset refs that
         # have been deleted already from registry but are still in
         # the python object. moveToTrash will deal with that.
-        self.bridge.moveToTrash([ref])
+        self.bridge.moveToTrash(refs)
```

Comment:
Might be time to just remove `_move_to_trash_in_registry` and call this method directly.